src/azure-iot-edge-installer.sh

```diff
@@ -12,3 +26,11 @@ chmod +x validate-post-install.sh
 ./install-container-management.sh
 ./install-edge-runtime.sh
 ./validate-post-install.sh
+cd ..
```

Comment:
might be better (safer?) to store the `cwd` in a local variable and cd to $(local variable) instead?